queries.sql

```diff
@@ -228,3 +228,144 @@ GROUP BY
 ORDER BY
     animal_count DESC
 LIMIT 1;
+
+
+--Write queries to answer the following:
+-- Who was the last animal seen by William Tatcher?
+SELECT
+    a.name AS last_animal_seen
+FROM
+    visits v
+JOIN
+    animals a ON v.animal_id = a.id
+WHERE
+    v.vet_id = (SELECT id FROM vets WHERE name = 'William Tatcher')
+ORDER BY
+    v.visit_date DESC
+LIMIT 1;
+
+
+-- How many different animals did Stephanie Mendez see?
+SELECT
+    COUNT(DISTINCT v.animal_id) AS number_of_animals_seen
+FROM
+    visits v
+WHERE
+    v.vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
+
+    
+-- List all vets and their specialties, including vets with no specialties.
+SELECT
+    v.name AS vet_name,
+    COALESCE(s.name, 'No Specialty') AS specialty_name
+FROM
+    vets v
+LEFT JOIN
+    specializations sp ON v.id = sp.vet_id
+LEFT JOIN
+    species s ON sp.species_id = s.id
+ORDER BY
+    v.name, specialty_name;
+
+
+-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+SELECT
+    a.name AS animal_name
+FROM
+    visits v
+JOIN
+    animals a ON v.animal_id = a.id
+WHERE
+    v.vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez')
+    AND v.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
```

Comment:
- There are two visits to Stephanie Mendez from April 1st and August 30th, 2020. Those are 👇🏽 

> Agumon visited Stephanie Mendez on Jul 22th, 2020.
> Blossom visited Stephanie Mendez on May 24th, 2020.

Please 🙏🏽 fix this query to have the correct information.